rfc6962-bis.xml

```diff
@@ -1195,7 +1203,34 @@ entry specified by <spanx style="verb">start</spanx>.
         </t>
       </section>
-      <section title="TLS Extension" anchor="tls_transinfo_extension">
+      <section title="Presenting SCTs, inclusion proofs and STHs" anchor="sct_with_proof">
+        <figure>
+          <preamble>
+            When constructing a <spanx style="verb">TransItemList</spanx> structure, a TLS server SHOULD construct and include <spanx style="verb">TransItem</spanx> structures of type <spanx style="verb">x509_sct_with_proof_v2</spanx> (for an SCT of type <spanx style="verb">x509_sct_v2</spanx>) or <spanx style="verb">precert_sct_with_proof_v2</spanx> (for an SCT of type <spanx style="verb">precert_sct_v2</spanx>), both of which encapsulate a <spanx style="verb">SCTWithProofDataV2</spanx> structure:
+          </preamble>
+          <artwork>
+    struct {
+        SignedCertificateTimestampDataV2 sct;
+        SignedTreeHeadDataV2 sth;
+        InclusionProofDataV2 inclusion_proof;
+    } SCTWithProofDataV2;</artwork>
+        </figure>
+        <t>
+          <spanx style="verb">sct</spanx> is the encapsulated data structure from an SCT that corresponds to the server certificate or to a name-constrained intermediate the server certificate chains to.
+        </t>
+        <t>
+          <spanx style="verb">sth</spanx> is the encapsulated data structure from an STH that was signed by the same log as <spanx style="verb">sct</spanx>.
+        </t>
+        <t>
+          <spanx style="verb">inclusion_proof</spanx> is the encapsulated data structure from an inclusion proof that corresponds to <spanx style="verb">sct</spanx> and can be used to compute the root in <spanx style="verb">sth</spanx>.
+        </t>
+      </section>
+      <section title="Presenting SCTs only">
+        <t>
+          Presenting inclusion proofs and STHs in the TLS handshake helps to protect the client's privacy (see <xref target="validating_inclusion_proofs"/>) and reduces load on log servers. However, if a TLS server is unable to obtain an inclusion proof and STH that correspond to an SCT, then it MAY include <spanx style="verb">TransItem</spanx> structures of type <spanx style="verb">x509_sct_v2</spanx> or <spanx style="verb">precert_sct_v2</spanx> in the <spanx style="verb">TransItemList</spanx>.
+        </t>
+      </section>
+      <section title="transparency_info TLS Extension" anchor="tls_transinfo_extension">
         <t>
           Provided that a TLS client includes the <spanx style="verb">transparency_info</spanx> extension type in the ClientHello, the TLS server MAY include the <spanx style="verb">transparency_info</spanx> extension in the ServerHello with <spanx style="verb">extension_data</spanx> set to a <spanx style="verb">TransItemList</spanx>. The TLS server SHOULD ignore any <spanx style="verb">extension_data</spanx> sent by the TLS client. Additionally, the TLS server MUST NOT process or include this extension when a TLS session is resumed, since session resumption uses the original session information.
```

Comment:
To capture an offline discussion: Since there's a MUST earlier the MAY would be changed to a SHOULD (and it fits with the paragraph's context).